.github/workflows/int-and-connected-test-run.yml

```
@@ -73,8 +73,8 @@ jobs:
           # extract service account credentials
           base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
           base64 --decode <<< ${{ secrets.B64_RBS_APPLICATION_CREDENTIALS }} > ${RBS_CLIENT_CREDENTIAL_FILE_PATH}
-          # assemble code and run connected tests
-          ./gradlew assemble -w testConnected --scan
+          # run connected tests
+          ./gradlew --scan -w testConnected
```

```suggestion
          ./gradlew --scan --warn testConnected
```

For clarity, could we switch these to the long version of the option?

---

Oh ditto this :) I was wondering what this flag meant but forgot to ask.